prometheus-exporters/snmp-ntp-exporter/alerts/snmp-ntp-cisco-n7k.alerts

```diff
@@ -1,16 +1,16 @@
 groups:
```

Comment:
Empty groups don't make sense. Either comment this one or remove the file entirely.

Reply:
I would leave the tier for now, since it does no harm. 
true at chart .. I forgot .. thanks.
will do.